src/js/components/Products/Product.js

```diff
@@ -0,0 +1,187 @@
+import React, { Component } from "react";
+import Navbar from "../Navbar/Navbar";
+import "./Product.scss";
+class Product extends Component {
+    constructor(props){
+        super(props);
+        this.state ={
+            Name:'',
+            CoverImageLocation:'',
+            Description:'',
+            Price:'',
+            Type:'',
+            SizeOption:'',
+            ColourOption:'' ,
+            small:false,
+            medium:false,
+            large:false,
+            Speedy:false,
+            color:false,
+            kyn:false,
+            count:0
+
+        }
+        this.onChange=this.onChange.bind(this);
+        this.onSubmit=this.onSubmit.bind(this);
+        this.Small=this.Small.bind(this);
+        this.Medium=this.Medium.bind(this);
+        this.Large=this.Large.bind(this);
+        this.handleChecked=this.handleChecked.bind(this);
+        this.Color=this.Color.bind(this)
+    }
+    onChange(e){
```

Comment:
Just add a space in between these function declarations and look a the scss file the indentations and empty lines spaces are inconsistent 

URLs should be added to const variables and be kept in a const file

Reply:
I created constants file for product and in that file, there is  const variable for url. I imported constants file on product and use the const variable for url instead of actual url.